### ISSUE TITLE
Keep draw after reload page

### ISSUE THREAD

**Issue by paulodfavero** [amcharts/amcharts5#1805]
How can I keep draw after reload page?
I saw that exist an object for each draw. Is it possible to use this data to plot in the chart again?

<img width="751" alt="image" src="https://github.com/user-attachments/assets/3b8833c0-4de6-439e-bdc9-a79c4277d1d3" />

**Comment by martynasma** [comment]
Yes, you can either use `drawingsupdated` event to grab current serialized drawings, and then restore them when needed.

Docs:
https://www.amcharts.com/docs/v5/charts/stock/serializing-indicators-annotations/

Or you can instruct your users to enable autosaving of drawings via settings or data save controls:

https://www.amcharts.com/docs/v5/charts/stock/toolbar/settings-control/#Auto_saving_feature
https://www.amcharts.com/docs/v5/charts/stock/toolbar/data-save-control/

**Comment by paulodfavero** [comment]
Thanks again! It helps a lot
